services/order.py

```diff
@@ -0,0 +1,51 @@
+from django.contrib.auth import get_user_model
+from db.models import MovieSession, Ticket, Order
+from django.db import transaction
+from django.db.models import QuerySet
+
+
+def validate_movie_session(movie_session_id: int) -> bool:
+    try:
+        MovieSession.objects.get(id=movie_session_id)
+    except MovieSession.DoesNotExist:
+        raise MovieSession.DoesNotExist(
+            f"MovieSession with id: {movie_session_id}, does not exist"
+        )
+
+    return True
+
+
+def create_order(
+        tickets: list[Ticket],
+        username: str,
+        date: str = None
+) -> None:
+
+    current_user = get_user_model().objects.get(username=username)
+    order = Order(user=current_user)

```

Comment:
number of queries remain the same, there is a use case for creating additional variable if you are referencing to it later, but here its just an additional line of code)

Reply:
sorry I misunderstood, now it is ok, I think)